docs/config-olap-databases.md

```diff
@@ -4,26 +4,38 @@ title: OLAP Database Configuration
 sidebar_label: OLAP Database
 ---
 
-## Section
+## Introduction
 
-`olap_database:` The OLAP (online analytical processing) database section configures the OLAP database that BuildBuddy uses to enable the Trends page. **Optional**
+To speed up the analysis of historical build event data, BuildBuddy relies on ClickHouse, an OLAP database solution.
+
+Deploying ClickHouse is optional when using BuildBuddy
+BuildBuddy does not require ClickHouse for its core features, such as Build Event Service, Remote Cache, and Remote Execution.
 
-Note: in order to use OLAP database for the Trends page, `app.enable_read_from_olap_db` and
-`app.enable_write_to_olap_db` needs to be set to `true`
+However, some UI features, such as Trends, Drilldown, and Test Grid, do rely on ClickHouse.
+Without a configured ClickHouse instance, these features will either not be displayed on our UI or will operate in a different mode.
 
 ## Options
 
 **Optional**
 
+`olap_database:` The OLAP (online analytical processing) database section configures the OLAP database that BuildBuddy uses to enable the Trends page. **Optional**
+
 - `data_source` This is a connection string used by the database driver to connect to the database. ClickHouse database is supported.
-- `enable_data_replication` If true, data replication is enabled.
+
+- `enable_data_replication` If ClickHouse is using a [Cluster Deployment](https://clickhouse.com/docs/en/architecture/cluster-deployment), this will enable data replication within the cluster.
```

Comment:
```suggestion
- `enable_data_replication` If ClickHouse is using a [cluster deployment](https://clickhouse.com/docs/en/architecture/cluster-deployment), this will enable data replication within the cluster.
```